tasks/src/fill-mask/about.md

```diff
@@ -1,14 +1,18 @@
+## Use Cases
 
-For example, masked language modeling is used to train large models for domain-specific problems. If you have to work on a domain-specific task such as information retrieval from medical research papers, you can train a masked language model using those papers. 📄
+### Domain Adaptation 👩‍⚕️
 
-The resulting model has a statistical understanding of the language used in medical research papers, and it can be trained further in a process called fine-tuning to solve different tasks, such as [Text Classification](/tasks/text-classification) or [Question Answering](/tasks/question-answering) to build a medical research papers information extraction system. 👩‍⚕️ Pre-training on domain-specific data tends to yield better results (see [this paper](https://arxiv.org/abs/2007.15779) for an example).
+Masked language models do not require labelled data! They are trained by masking a couple of words in sentences and the model is expected to guess the masked word. This makes it very practical!
 
-If you don't have the data to train a masked language model, you can also use an existing [domain-specific masked language model](https://huggingface.co/microsoft/BiomedNLP-PubMedBERT-base-uncased-abstract-fulltext) from the Hub and fine-tune with your smaller task dataset. That's the magic of Open Source and sharing your work! 🎉
+For example, masked language modeling is used to train large models for domain-specific problems. If you have to work on a domain-specific task, such as retrieving information from medical research papers, you can train a masked language model using those papers. 📄
+
+The resulting model has a statistical understanding of the language used in medical research papers, and can be further trained in a process called fine-tuning to solve different tasks, such as [Text Classification](/tasks/text-classification) or [Question Answering](/tasks/question-answering) to build a medical research papers information extraction system. 👩‍⚕️ Pre-training on domain-specific data tends to yield better results (see [this paper](https://arxiv.org/abs/2007.15779) for an example).
+
+If you don't have the data to train a masked language model, you can also use an existing [domain-specific masked language model](https://huggingface.co/microsoft/BiomedNLP-PubMedBERT-base-uncased-abstract-fulltext) from the Hub and fine-tune it with your smaller task dataset. That's the magic of Open Source and sharing your work! 🎉
 
 ## Inference with Fill-Mask Pipeline
-You can use the 🤗 Transformers library `fill-mask` pipeline to do inference with masked language models. If a model name is not provided, the pipeline will be initialized with [distilroberta-base](/distilroberta-base). You can provide masked text, and it will return list of possible mask values, ranked according to the score.
+
```

Comment:
I'd rather not have any gaps.

Reply:
In terms of markdown it should not make any difference in rendering I think. The newline helps with readability in the code imo